src/collections/_documentation/accounts/quotas/index.md

```diff
@@ -0,0 +1,216 @@
+
+3. **Quota availability**
+
+      Events that exceed your quota are not sent. To add to your quota or review what happens when you exceed it, see [Increasing Quotas]({%- link _documentation/accounts/quotas/index.md -%}#increasing-quotas).
+
+4. **Event repetition**
+  -   If you have intervened to discard events with the same fingerprint, the event does not count toward your quota. For more information, see [Delete & Discard in our guide to Manage Your Event Stream]({%- link _documentation/accounts/quotas/manage-event-stream-guide.md  -%}#delete--discard) or [Inbound Filters]({%- link _documentation/accounts/quotas/index.md -%}#inbound-data-filters).
+  -   If the previous event was resolved, this event counts toward your quota because it may represent a regression in your code.
+  -   If you have intervened to ignore alerts about events with the same fingerprint, this event counts toward your quota because the event is still occurring. For more information, see [Inbound Filters]({%- link _documentation/accounts/quotas/index.md -%}#inbound-data-filters).
+
+5. **Spike protection**
+
+      Sentry’s spike protection prevents huge overages from consuming your event capacity. For more information, see [Spike Protection]({%- link _documentation/accounts/quotas/index.md -%}#spike-protection).
+
+In addition, depending on your project’s configuration and the plan you subscribe to, Sentry may also check:
+
+6. **Rate limit for the project**
+
+      If the rate limit for the project has been exceeded, and your subscription allows, the event will not be counted. For more information, see [Rate Limiting in our guide to Manage Your Event Stream]({%- link _documentation/accounts/quotas/manage-event-stream-guide.md -%}#4-rate-limiting) or  [Rate Limiting Projects]({%- link _documentation/accounts/quotas/index.md -%}#id1).
+
+7. **Inbound Filters**
+
+      If any inbound filter is set for this type of event, and your subscription allows, the event will not be counted. For more information, see [Inbound Filters in our guide to Manage Your Event Stream]({%- link _documentation/accounts/quotas/manage-event-stream-guide.md -%}#inbound-data-filters) or [Inbound Data Filter]({%- link _documentation/accounts/quotas/index.md -%}#inbound-data-filters).
+
+After these checks are processed, the event counts toward your quota. It is accepted into Sentry, where it persists and is stored.
+
+A few important notes:
+  - For mobile users, a storage fee is also charged if the Native SDK minidump is attached to the event in addition to the event counting toward your quota. For more information, see [Event Attachments (Preview)]({%- link _documentation/platforms/native/minidump.md -%}#event-attachments-preview).
```

Comment:
This is not a mobile use case. You can just mention: 

> A storage fee is charged for minidumps and event attachments on stored events. For more information, ...

Also, note that this is not GA yet and subject to special sales agreements, until we roll out the extended quotas for attachment storage (after which the above definition of quota no longer holds, as it also applies to non-event items).

Reply:
I will make the edit to clarify it's true for all, not just mobile. But since it is not yet GA, would you like this commented out for now?

Reply:
Commenting it out would be better for now, yes.

Reply:
Done. Thanks.